<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>686</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Автомат</string>
  </property>
  <property name="windowIcon">
   <iconset theme="D:\Dropbox\GitHub\visualization_of_automata\icons\window_icons\python.png"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="12" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="start_vizualization">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Визуализировать</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="10">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1">
     <layout class="QVBoxLayout" name="states_label_layout">
      <item>
       <widget class="QLabel" name="states_label">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Таблица переходов</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="alphabet_layout">
      <item>
       <widget class="QLabel" name="alphabet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="1">
     <layout class="QVBoxLayout" name="word_input_layout">
      <item>
       <widget class="QTextEdit" name="word_input">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Введите слово</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="10">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="alphabet_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Алфавит</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="automat_name_layout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="automata_name">
        <property name="minimumSize">
         <size>
          <width>498</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1">
     <layout class="QVBoxLayout" name="table_layout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
    <item row="5" column="1">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_top_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openfile_top_menu"/>
   </widget>
   <widget class="QMenu" name="create_top_menu">
    <property name="title">
     <string>Создать</string>
    </property>
    <addaction name="create_DFA_top_menu"/>
    <addaction name="create_DFMA_top_menu"/>
   </widget>
   <addaction name="file_top_menu"/>
   <addaction name="create_top_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openfile_top_menu">
   <property name="text">
    <string>Открыть файл</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="create_DFA_top_menu">
   <property name="text">
    <string>Конечный детерминированный автомат</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="create_DFMA_top_menu">
   <property name="text">
    <string>Конечный детерминированный автомат с магазинной памятью</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
